src/test/resources/null

```diff
@@ -0,0 +1 @@
+{"Vehicle":{"Car":"Blue Tacoma"}}
```

Comment:
I think you will want to add this file path to .gitignore so you don't have to add new files every commit.